app/helpers/invoices_helper.rb

```diff
@@ -15,6 +15,14 @@ def get_address(invoice)
     end
   end
 
+  def get_address_first(client)
```

Comment:
`get_address` and `get_address_first` is doing same thing? Can we reuse this method?